### PR TITLE
fix(discussions): only show replies form if discussion is not closed

### PR DESCRIPTION
**Pull request by hypeJunction**
Makes sure discussion reply form is not shown if the discussion is closed.
In river views, makes sure user has write permissions.
- [ ] Update once #9751 is merged

### CODE REVIEW

**Comment by jdalsem** [comment]
LGTM

**Comment by hypeJunction** [comment]
Added 3 more commits here.

File: mod/discussions/views/default/river/object/discussion/create.php [hunk 1]
```diff
@@ -3,17 +3,18 @@
...
 
 $object = $item->getObjectEntity();
 $excerpt = strip_tags($object->description);
 $excerpt = elgg_get_excerpt($excerpt);
 
-$responses = '';
-if (elgg_is_logged_in() && $object->canWriteToContainer()) {
-	$responses = elgg_view('river/elements/discussion_replies', array('topic' => $object));
-}
+$responses = elgg_view('river/elements/discussion_replies', array('topic' => $object));
```
Comment (hypeJunction) [inline mod/discussions/views/default/river/object/discussion/create.php]: write permissions are checked before displaying a form

**Comment by hypeJunction** [comment]
Move the status check to the `container_permissions_check` hook, as we end duplicating all the checks all over again.

**Comment by jdalsem** [comment]
I believe you can finish this after a rebase

**Comment by hypeJunction** [comment]
I need to redo this now that the container logic check is merged

**Comment by hypeJunction** [comment]
Replaced by #10021

**Pull request closed without merging**